app/templates/home.hbs

```diff
@@ -1,83 +1,61 @@
 <div class="home-page">
```

Comment:
This whole file seems to be 4-space-indented. Probably should be consistent between the JS and HTML style and use 2 spaces everywhere.